### PR TITLE
Update footer links

### PR DESCRIPTION
**Pull request by KellyStathis**
## Purpose

Updates the site footer for https://commons.datacite.org/ to correspond with the new website structure.

## Approach

- Updated Footer.tsx to match the new footer links structure, per the list in [DataCite - homepage inventory](https://docs.google.com/spreadsheets/d/1Ei6A3RfYuNmNvquq5W8TGeyf5C9PEz7F7HmM-6Taba8/edit#gid=1575567007). This matches the list in links.json (from assets repository) but does not use this file.
- Reordered social links and added the Mastodon link.

#### Open Questions and Pre-Merge TODOs

- [ ] With a more recent version of fort awesome we can use faXTwitter (see https://www.npmjs.com/package/@fortawesome/free-brands-svg-icons?activeTab=code) instead of the svg. However, we are currently on 5.15.3 which does not include faXTwitter. If it is simpler to upgrade the version, that would be a cleaner solution than the svg.
- [ ] There is an open todo in Footer.tsx to "Links from '../../../config/links.yml'. I haven't updated links.yml yet. Is this file used and if so should I update it?

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)

- [x] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Reviewer, please remember our [guidelines](https://datacite.atlassian.net/wiki/spaces/TEC/pages/1168375809/Pull+Request+Guidelines):

- Be humble in the language and feedback you give, ask don't tell.
- Consider using positive language as opposed to neutral when offering feedback. This is to avoid the negative bias that can occur with neutral language appearing negative.
- Offer suggestions on how to improve code e.g. simplification or expanding clarity.
- Ensure you give reasons for the changes you are proposing.

### CODE REVIEW

**Comment by KellyStathis** [comment]
@jrhoads, Suzanne mentioned you might be a good person to ask about the open questions, since you worked on this recently

**Review by jrhoads — APPROVED** [review]

**Review by jrhoads — APPROVED** [review]

**Review by svogt0511 — APPROVED** [review]
Looks good to me!  Glad you were able to upgrade over using the SVGs.  Sorry I couldn't do the review earlier.  

@KellyStathis or @jrhoads - were one of you going to merge this or should I? (Re: the Thursday deadline Paul mentioned.)

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -105,44 +103,47 @@ const Footer = () => {
...
               <FontAwesomeIcon icon={faBlog} />
             </a>
-            <a href="https://twitter.com/datacite" className="share">
-              <FontAwesomeIcon icon={faTwitter} />
-            </a>
             <a href="https://github.com/datacite/datacite" className="share">
               <FontAwesomeIcon icon={faGithub} />
             </a>
+            <a href="https://twitter.com/datacite" className="share">
+              <svg aria-hidden="true" focusable="false" data-prefix="fab" class="svg-inline--fa fa-twitter" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path fill="currentColor" d="M389.2 48h70.6L305.6 224.2 487 464H345L233.7 318.6 106.5 464H35.8L200.7 275.5 26.8 48H172.4L272.9 180.9 389.2 48zM364.4 421.8h39.1L151.1 88h-42L364.4 421.8z"></path></svg>
```
Comment (jrhoads) [inline src/components/Footer/Footer.tsx:131]: This should probably have `className` instead of `class`

**Comment by svogt0511** [comment]
Fixed the warnings in the browser dev console:

- I updated the <svg> changing "class" to "className" in src/components/Footer/Footer.tsx
- Changed <p></p> that enclosed the examples in all of the tab index.tsx files to <div></div>  (see the screengrab)

9A2728B9-5550-4407-A830-4FE128A0A33A

I am closing this pull request and deploying it to production.

**Pull request merged**